golden_response/staging/get_triples/place_type.json

```diff
@@ -28,7 +28,7 @@
       "objectId": "Country",
       "objectName": "Country",
       "objectTypes": ["Class"],
-      "provenanceId": "dc/5j06ly1"
+      "provenanceId": "dc/5n63hr1"
```

Comment:
@tjann provenance change here

Reply:
Went from WikidataGeos to WikidataOtherIdGeos. Seems reasonable for now.